modules/issue_tracker/jsx/issueTrackerIndex.js

```diff
@@ -0,0 +1,250 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import Loader from 'Loader';
+import FilterableDataTable from 'FilterableDataTable';
+
+class IssueTrackerIndex extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data: {},
+      error: false,
+      isLoaded: false,
+    };
+
+    this.fetchData = this.fetchData.bind(this);
+    this.formatColumn = this.formatColumn.bind(this);
+  }
+
+  componentDidMount() {
+    this.fetchData()
+      .then(() => this.setState({isLoaded: true}));
+  }
+
+  /**
+   * Retrieve data from the provided URL and save it in state
+   * Additionally add hiddenHeaders to global loris variable
+   * for easy access by columnFormatter.
+   *
+   * @return {object}
+   */
+  fetchData() {
+    return fetch(this.props.dataURL, {credentials: 'same-origin'})
+      .then((resp) => resp.json())
+      .then((data) => this.setState({data}))
+      .catch((error) => {
+        this.setState({error: true});
+        console.error(error);
+      });
+  }
+
+  /**
+   * Modify behaviour of specified column cells in the Data Table component
+   *
+   * @param {string} column - column name
+   * @param {string} cell - cell content
+   * @param {object} row - row content indexed by column
+   *
+   * @return {*} a formated table cell for a given column
+   */
+  formatColumn(column, cell, row) {
+    let result = <td>{cell}</td>;
+    let link;
+    switch (column) {
+    case 'Title':
+      link = (
+        <a
+          href={loris.BaseURL+'/issue_tracker/issue/?issueID='+row['Issue ID']}
```

Comment:
I think these URLs should be `/issue_tracker/issues/+row['Issue ID']`, not via a query parameter.

Reply:
This is still being done through the old ajax call system. IMO I would change this with a PR that creates a real LORIS endpoint for this request.